helm/promtail/templates/service-metrics.yaml

```diff
@@ -14,4 +14,3 @@ spec:
       protocol: TCP
   selector:
     {{- include "promtail.selectorLabels" . | nindent 4 }}
-{{- end }}
```

Comment:
Why do you want to remove the toggle? This is a deviation from upstream and changing the default config in values.yaml would make more sense IMO.

Reply:
This is indeed a deviation! Setting `serviceMonitor.enabled` to `true` in the default values would try to install the `ServiceMonitor` CR in file [templates/servicemonitor.yaml](https://github.com/giantswarm/promtail-app/blob/master/helm/promtail/templates/servicemonitor.yaml) though which requires the Prometheus CRDs installed on the cluster.